examples/specs/selection_type_point_zorder.vl.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
+  "data": {
+    "url": "https://cdn.jsdelivr.net/npm/vega-datasets@v1.29.0/data/cars.json"
+  },
+  "mark": {"type": "circle", "opacity": 1},
+  "encoding": {
+    "color": {
+      "condition": {"param": "param_122", "value": "coral", "empty": false},
+      "value": "grey"
+    },
+    "order": {
+      "value": 0,
+      "condition": {"param": "param_122", "value": 1, "empty": false}
```

Comment:
For consisistency with size and to make it reads like if-then-else, better put `condition` above `value`.